### PR TITLE
Add django-sslify middleware

### LINKED ISSUE #82: Add redirect if hitting the app on http:// instead of https://
I think there is native support for this in django 1.9

### PR DESCRIPTION
**Pull request by justinabrahms**
This will protect against hitting the heroku instance over http instead
of https.

See also, https://github.com/mitodl/teachersportal/pull/171

Fixes #82

### CODE REVIEW

File: courses/views_test.py [hunk 1]
```diff
@@ -37,7 +40,7 @@ def test_separate_children(self):
         # Different courses
         self.assertNotEqual(m1.course, m2.course)
 
-        resp = self.client.get(reverse('module-list', args=(m1.course.uuid,)))
+        resp = self.client.get(reverse('module-list', args=(m1.course.uuid,)), **self.https)
```
Comment (noisecapella) [inline courses/views_test.py]: Why not make `SSLIFY_DISABLE` an environment variable and then disable it for all tests instead?

**Comment by justinabrahms** [comment]
Updated to use envvar.

**Comment by noisecapella** [comment]
:+1:

**Comment by bdero** [comment]
This was already solved by #86 . It looks like I forgot to close the issue, sorry about that. http://ccxcon-ci.herokuapp.com 301's to https.

**Comment by justinabrahms** [comment]
We should do this same fix on TP then.

**Comment by justinabrahms** [comment]
(if we can. I know it's on django 1.8, not 1.9. Not sure when that setting got added)

**Comment by bdero** [comment]
Yeah, I just finished fixing the PR for it on TP: https://github.com/mitodl/teachersportal/pull/136 It's ready to go, just wanted 1 final thumbs up before I hit merge.

**Pull request closed without merging**